src/main/java/org/opensearch/neuralsearch/transport/MLPredictAction.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.neuralsearch.transport;
+
+import org.opensearch.action.ActionType;
+import org.opensearch.common.io.stream.Writeable;
+
+public class MLPredictAction extends ActionType<MLPredictActionResponse> {
+
+    public static final MLPredictAction INSTANCE = new MLPredictAction();
+    public static final String NAME = "cluster:admin/ml_predict_action";
```

Comment:
This short action may conflict with others, how about use a longer name with your plugin name like 
`cluster:admin/opensearch/neural-search/ml_predict_action`

Reply:
Yeah sure make sense. Let me update this.